src/index.js

```diff
@@ -219,7 +214,7 @@ MyArray.prototype.slice = function(beginArg, endArg) {
 MyArray.prototype[Symbol.toPrimitive] = function(hint) {
   switch (hint) {
   case 'string':
-    return `Array has ${this.length} elements`;
+    return this.toString();
```

Comment:
Was it mistake? I thought we may use any values for STRING and NUMBER. It just depends on what we want to get as a string or as a value.
Who forced you to change it? ))

Reply:
Actually it was Timophey)
He said that string case of toPrimitive should had behavior like toString

Reply:
Sounds weird ) Ok